schema.sql

```diff
@@ -44,4 +44,13 @@ CREATE TABLE visits (
 animals_id int references animals(id),
 vets_id int references vets(id),
 date_of_visit date
-);
\ No newline at end of file
+);
+
+ALTER TABLE visits RENAME COLUMN animals_id to animal_id;
+ALTER TABLE visits RENAME COLUMN vets_id to vet_id;
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+ALTER TABLE visits ADD COLUMN id serial primary key;
+
+CREATE INDEX ON visits(animal_id);
+CREATE INDEX ON visits(vet_id);
```

Comment:
- [ ] Please add the missed `CREATE INDEX ON owners(email);`
_Don't forget to update the screenshots in the PR summary after that._ 😉